extensions/sql-assessment/src/tabs/assessmentMainTab.ts

```diff
@@ -96,8 +95,14 @@ export class SqlAssessmentMainTab extends SqlAssessmentTab {
 		];
 	}
 
-	private async createToolbar(view: azdata.ModelView): Promise<azdata.ToolbarContainer> {
+	private limitLongName(name: string): string {
```

Comment:
Side note - but functions like this generally are better put as standalone functions at the module level instead of in a class since they don't actually need any class members.